manifests/g/GovernmentOfThePRC/ZheJiangProvince/ZheZhengDing/2.21.8/GovernmentOfThePRC.ZheJiangProvince.ZheZhengDing.locale.zh-CN.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: GovernmentOfThePRC.ZheJiangProvince.ZheZhengDing
+PackageVersion: 2.21.8
+PackageLocale: zh-CN
+Publisher: 杭州市西湖区浙江省数据局（浙江省政务服务管理办公室）
+PublisherUrl: https://www.zj.gov.cn/

```

Comment:
```suggestion
#PublisherUrl: https://www.zj.gov.cn/
```

![image](https://github.com/user-attachments/assets/97af23ca-adb0-4ee5-bc5d-c06d40d04831)
Can this be waived?

Reply:
If this url can be waived, I will add it back to the manifest later.